package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
```

Comment:
- [ ] Please change those command to the following:

```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

because it will separate between the development and the production mode

Reply:
Alright, right away.

Reply:
well resolved! ✔️